src/main/resources/Task 1 - Shoe Object Modelling.txt

```diff
@@ -0,0 +1,5 @@
+Object Modeling Shoes:
+- create a class diagram for shoes object hierarchy.
+- implement class diagram in Java code.
+- use static code analysis tools to verify code for issues and potential bugs.
```

Comment:
on diagram you are suppose to show objects blueprints: properties and methods. Constructors are not member of objects there fore should not be displayed on diagram.

Reply:
The constructors was deleted.